### ISSUE TITLE
Issues encountered should indicate whether the github issues have been closed

### ISSUE THREAD

**Issue by sbomer** [jaredpar/runfo#150]
For example, see the "Issues Encountered" here: https://runfo.azurewebsites.net/view/definition/1080/.
https://github.com/dotnet/runtime/issues/64280 has been closed but there is no indication of this in the runfo UI.

**Comment by jaredpar** [comment]
Agree. Need to begin listening to events for issues, tracking them and updating them in the UI.
